libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -0,0 +1,86 @@
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    Component,
+    ContentChild,
+    ContentChildren,
+    forwardRef,
+    Input,
+    QueryList,
+    ViewEncapsulation
+} from '@angular/core';
+import { FormControlDirective, TokenComponent } from '@fundamental-ngx/core';
+
+@Component({
+  selector: 'fd-tokenizer',
+  templateUrl: './tokenizer.component.html',
+  styleUrls: ['./tokenizer.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TokenizerComponent implements AfterViewInit {
+
+    @ContentChildren(forwardRef(() => TokenComponent))
+    tokenList: QueryList<TokenComponent>;
+
+    @ContentChild(forwardRef(() => FormControlDirective), {static: true})
+    input: FormControlDirective;
+
+    /** Used to add focus class to the tokenizer-example */
+    @Input()
+    tokenizerHasFocus: boolean = false;
+
+    /** Whether the tokenizer-example is compact */
+    @Input()
+    compact: boolean = false;
+
+    /** The value for the tokenizer input */
+    @Input()
+    inputValue: string;
+
+    /** @hidden */
+    ngAfterViewInit() {
+        this.input.elementRef.nativeElement.addEventListener('keydown', (event) => this.inputKeyDown(event));
+    }
+
+    /** @hidden */
+    inputKeyDown(event) {
+        if (event.key === 'ArrowLeft') {
+            this.handleTokenFocus(event, this.tokenList.length);
+        }
+    }
+
+    /** @hidden */
+    handleTokenFocus(event, fromIndex) {
+        let newIndex;
+        // function needs to be defined in order to be referenced later by addEventListener/removeEventListener
+        const handleFunctionReference = (e) => {
+            if (newIndex || newIndex === 0) {
+                this.handleTokenFocus(e, newIndex);
+            }
+        };
+        if (event.code === 'ArrowLeft' || event.code === 'ArrowRight') {
+            let elementToFocus;
+            if (event.code === 'ArrowLeft') {
+                newIndex = fromIndex - 1;
+            } else if (event.code === 'ArrowRight') {
+                newIndex = fromIndex + 1;
+            }
+            if (newIndex >= 0 && newIndex < this.tokenList.length) {
+                elementToFocus = this.tokenList.filter((element, index) => index === newIndex)[0]
+                    .elementRef.nativeElement.querySelector('.fd-token');
```

Comment:
Could you add `focus` method on `td-token` element and use elements from `contentChildren`, instead of queryList? 
Currently there is JS solution. I would in general replace it by observables with `renderer.listen` methods. You can check how it's done with in [tabs](https://github.com/SAP/fundamental-ngx/blob/26746d3fa15628b7be3aebf012bf696d762c271e/libs/core/src/lib/tabs/tab-nav/tab-nav.component.ts)
@salarenko  @rengare  @InnaAtanasova  What do you think about it?